packages/next/client/components/layout-router.tsx

```diff
@@ -80,8 +81,8 @@ function walkAddRefetch(
  * Wraps ReactDOM.findDOMNode with additional logic to hide React Strict Mode warning
  */
 function findDOMNode(
-  instance: Parameters<typeof ReactDOMfindDOMNode>[0]
-): ReturnType<typeof ReactDOMfindDOMNode> {
+  instance: Parameters<typeof TFindDOMNode>[0]
```

Comment:
You don't need `typeof` here right?

Reply:
ts doesn't allow me to do that so I use this. seems `TFindDOMNode` is just a alias to the original module. gonna change to `typeof React.findDOMNode` to reduce the confusion here